src/main/java/com/kodilla/ecommercee/repository/ProductRepository.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Product;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ProductRepository extends CrudRepository <Product, Long>{
+
+
```

Comment:
Tutaj też nadmiarowe puste linie. Usuń te puste linie we wszystkich plikach

Reply:
Stylistyka,  mergeujemy